src/hooks/useAuth.ts

```
@@ -1,25 +1,95 @@
   // 로그아웃 함수
   const logout = () => {
     localStorage.removeItem('access_token');
+    delete axiosInstance.defaults.headers.common['Authorization'];
     setIsAuthenticated(false);
   };
 
-  return { isAuthenticated, isLoading, logout };
+  // 토큰 재발급 함수
+  const refreshToken = async () => {
+    try {
+      const response = await axiosInstance.get('/api/jwt/access-token', {
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)